spec/models/manageiq/providers/kubernetes/container_manager/metrics_capture/capture_context_spec.rb

```diff
@@ -0,0 +1,90 @@
+describe ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCapture::CaptureContext do
+  before(:each) do
+    allow(MiqServer).to receive(:my_zone).and_return("default")
+    hostname = 'ose-master.qa.lab.com'
```

Comment:
Consider adding a comment what one would need to re-record this VCR.
Just any openshift env with working metrics?